examples/llm/src/mosaic_gpt.py

```diff
@@ -421,10 +421,14 @@ def forward(self,
                                     seq_len=S,
                                     key_padding_mask=key_padding_mask,
                                     dtype=x.dtype)
+        if self.cfg.attn_impl == 'flash' and key_padding_mask is None:
+            mod_key_padding_mask = torch.ones_like(input_ids, dtype=torch.bool)
```

Comment:
A small comment here on why this is necessary (may be with PR link) will be useful for anyone reading this in future.